ports/demonstone64/port.json

```diff
@@ -1,29 +1,30 @@
 {
-    "version": 2,
-    "name": "demonstone64.zip",
-    "items": [
-        "Demonstone64.sh",
-        "demonstone64"
+  "version": 3,
+  "name": "demonstone64.zip",
+  "items": [
+    "Demonstone64.sh",
+    "demonstone64"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Demonstone64",
+    "porter": [
+      "MadShmupper"
```

Comment:
You remofved jan from porters, this would wipe the portrers history of that port.